### ISSUE TITLE
ReST markup error in servers.rst

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7636]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)| @khorn reported|
|-|-|
|Trac ID|trac#7636|
|Type|enhancement|
|Created|2014-09-14 21:39:09Z|
|Branch|https://github.com/twisted/twisted/tree/markup-error-7636|

There is a minor markup error in `docs/core/howto/servers.rst`.

This inline markup:

```
:api:`twisted.protocols.basic.LineReceiver <LineReceiver>``
```

has an extra backtick.

This ticket should fix that.

<details><summary>Searchable metadata</summary>

```
trac-id__7636 7636
type__enhancement enhancement
reporter__khorn khorn
priority__normal normal
milestone__ 
branch__branches_markup_error_7636 branches/markup-error-7636
branch_author__khorn khorn
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation_easy documentation easy
time__1410730749713760 1410730749713760
changetime__1410732821427191 1410732821427191
version__None None
owner__khorn khorn

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_1"></a>|@khorn commented|
|-|-|

(In [43066]) Branching to markup-error-7636.

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_2"></a>|@khorn commented|
|-|-|

(In [43067]) Fix minor markuop error in docs/core/howto/servers.rst.  Refs [#7636](https://github.com/twisted/twisted/issues/7636)

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)|@khorn set owner to @glyph|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_4"></a>|@khorn commented|
|-|-|

Assigning to glyph, as it's not currently possible to assign to no one.

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|Alex set owner to @khorn|
|-|-|

Please add a news fragment (`.misc` is fine) and merge, this LGTM.

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_6"></a>|@khorn set status to `closed`|
|-|-|

(In [43072]) Merge markup-error-7636: Fix markup error in docs/core/howtos/servers.rst

Author: khorn
Reviewers: Alex
Fixes: [#7636](https://github.com/twisted/twisted/issues/7636)

Fix a minor markup error in docs/core/howtos/servers.rst
